README.md

```diff
@@ -28,6 +28,13 @@ apt-get -y install ruby-kramdown-rfc2629 python3-pip git
 pip3 install xml2rfc
 ```
 
+Installing cddl tool https://rubygems.org/gems/cddl
+```
+gem install cddl
```

Comment:
After running this command, I get

"/var/lib/gems/2.7.0/gems/cddl-0.9.1/lib/cddl.rb:1262:in `type_recall': Unknown type suit-reference nil [{}, {}, {}, {}, {}] (RuntimeError)"

Reply:
I was able to run this in my Mac laptop (MACOS 12.6), and also earlier version by -v option. My Mac has Ruby Gems v2.6.0.

BTW, the validation of CBOR files from the Makefile generates an error in using SUIT_*, which isn't in scope for this PR to fix. We need a different PR to fix the schema reference to SUIT.

cddl check-draft-ietf-teep-protocol.cddl validate ../cbor/query_request.diag.bin || exit 1; cddl check-draft-ietf-teep-protocol.cddl validate ../cbor/query_response.diag.bin || exit 1; cddl check-draft-ietf-teep-protocol.cddl validate ../cbor/update.diag.bin || exit 1; cddl check-draft-ietf-teep-protocol.cddl validate ../cbor/teep_success.diag.bin || exit 1; cddl check-draft-ietf-teep-protocol.cddl validate ../cbor/teep_error.diag.bin || exit 1;
/Library/Ruby/Gems/2.6.0/gems/cddl-0.9.1/lib/cddl.rb:1077:in **`rule_lookup': $$SUIT_Parameters not a type** [:grpchoice, [[:member, 1, 1, [:int, 1], [:type1, [:anno, :size, [:prim, 2], [:int, 16]], [:prim, 6, 112, [:prim, 2]]]]], [[:member, 1, 1, [:int, 2], [:anno, :size, [:prim, 2], [:int, 16]]]], [[:member, 1, 1, [:int, 3], [:anno, :cbor, [:prim, 2], [:array, [:member, 1, 1, [:text, "suit-digest-algorithm-id"], [:type1, [:int, -16], [:int, -18], [:int, -43], [:int, -44], [:int, -45]]], [:member, 1, 1, [:text, "suit-digest-bytes"], [:prim, 2]]]]]], [[:member, 1, 1, [:int, 14], [:prim, 0]]], [[:member, 1, 1, [:int, 5], [:prim, 0]]], [[:member, 1, 1, [:int, 21], [:prim, 3]]], [[:member, 1, 1, [:int, 22], [:prim, 0]]], [[:member, 1, 1, [:int, 23], [:prim, 2]]], [[:member, 1, 1, [:int, 24], [:anno, :size, [:prim, 2], [:int, 16]]]], [[:member, 1, 1, [:prim, 1], [:type1, [:type1, [:prim, 0], [:prim, 1]], [:type1, [:prim, 7, 20], [:prim, 7, 21]], [:prim, 3], [:prim, 2]]]], [[:member, 1, 1, [:int, 18], [:prim, 2]]], [[:member, 1, 1, [:int, 12], [:type1, [:prim, 7, 20], [:prim, 7, 21]]]], [[:member, 1, 1, [:int, 13], [:type1, [:prim, 7, 20], [:prim, 7, 21]]]]] (**RuntimeError**)
	from /Library/Ruby/Gems/2.6.0/gems/cddl-0.9.1/lib/cddl.rb:1259:in `type_recall'

Reply:
I got the same error Dave you got in cddl validation after I updated the suit-report cddl to its latest spec.

cddl check-draft-ietf-teep-protocol.cddl validate ../cbor/query_request.diag.bin || exit 1; cddl check-draft-ietf-teep-protocol.cddl validate ../cbor/query_response.diag.bin || exit 1; cddl check-draft-ietf-teep-protocol.cddl validate ../cbor/update.diag.bin || exit 1; cddl check-draft-ietf-teep-protocol.cddl validate ../cbor/teep_success.diag.bin || exit 1; cddl check-draft-ietf-teep-protocol.cddl validate ../cbor/teep_error.diag.bin || exit 1;
/Library/Ruby/Gems/2.6.0/gems/cddl-0.9.1/lib/cddl.rb:1262:in `type_recall': Unknown type suit-reference nil [{}, {}, {}, {}, {}] (RuntimeError)